utils/getInvoices/tests/getInvoices.electricity.test.ts

```diff
@@ -6,143 +6,551 @@ import { ServiceType } from '@utils/constants'
+        electricityPrice: undefined,
+      }
+
+      const invoices = getInvoices({
+        service,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
     })
   })
 
-  it('should load Electricity with previous month from service and prevPayment', () => {
-    const company: Partial<IRealestate> = null
-    const service: Partial<IService> = {
-      electricityPrice: 123,
-    }
-    const payment: Partial<IPayment> = null
-    const prevPayment: Partial<IPayment> = {
-      invoice: [
-        {
-          type: ServiceType.Electricity,
-          amount: 12,
-          lastAmount: 1,
-          price: 10,
-          sum: 110,
-        },
-      ],
-    }
-
-    const invoices = getInvoices({
-      company,
-      service,
-      payment,
-      prevPayment,
-    })
-
-    expect(invoices).toContainEqual({
-      type: ServiceType.Electricity,
-      amount: prevPayment.invoice[0].amount,
-      lastAmount: prevPayment.invoice[0].amount,
-      price: service.electricityPrice,
-      sum: 0,
+  describe('props: { company }', () => {
+    it('should NOT load when company = null', () => {
+      const company: Partial<IRealestate> = null
+
+      const invoices = getInvoices({
+        company,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+
+    it('should NOT load when company = { ...: 0 }', () => {
+      const company: Partial<IRealestate> = {
+        discount: 0,
+      }
+
+      const invoices = getInvoices({
+        company,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+    
+    it('should NOT load when company = { ...: 10 }', () => {
+      const company: Partial<IRealestate> = {
+        discount: 10,
+      }
+
+      const invoices = getInvoices({
+        company,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+
+    it('should NOT load when company = { ...: NaN }', () => {
+      const company: Partial<IRealestate> = {
+        discount: NaN,
+      }
+
+      const invoices = getInvoices({
+        company,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+
+    it('should NOT load when company = { ...: null }', () => {
+      const company: Partial<IRealestate> = {
+        discount: null,
+      }
+
+      const invoices = getInvoices({
+        company,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+
+    it('should NOT load when company = { ...: undefined }', () => {
+      const company: Partial<IRealestate> = {
+        discount: undefined,
+      }
+
+      const invoices = getInvoices({
+        company,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
```

Comment:
Join this all to `should NOT load when company = { ... }`. Absolutely not related to what the company contains though